mnemosyne/dr/recovery.py

```
@@ -163,7 +163,7 @@ def emergency_restore(backup_dir: Path = None, db_path: Path = None) -> Dict:
                     "backup_used": str(backup),
                     "attempts": 1
                 }
-        except Exception as e:
+        except Exception:
             continue
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)